firecrown/models/ccl_density.py

```diff
@@ -0,0 +1,108 @@
+"""Critical and Mean Density  Mass Module - Implementation of the NumberDensity
+abstract class for a critical and mean mass definition.
+========================================
+Class for mass definitions with Delta=200 times the critical/mean density.
```

Comment:
Why restrict things to Delta=200?

Reply:
I created this first density object implementation from the examples I was running with cosmosis. I can make an `overdensity` option  or `delta` option in the class init to make more generic. What do you think? I can let a default implementation with `delta=200` in case the user does not provide it.